crates/router/src/core/admin.rs

```diff
@@ -1918,7 +1924,53 @@ impl<'a> MerchantDefaultConfigUpdate<'a> {
         Ok(())
     }
 }
-
+#[cfg(all(
+    feature = "v2",
+    feature = "routing_v2",
+    feature = "business_profile_v2"
+))]
+struct MerchantDefaultConfigUpdate<'a> {
+    routable_connector: &'a Option<api_enums::RoutableConnectors>,
+    merchant_connector_id: &'a String,
+    state: &'a SessionState,
+    merchant_id: &'a id_type::MerchantId,
+    default_routing_config_for_profile: &'a Vec<api_models::routing::RoutableConnectorChoice>,
+    business_profile: BusinessProfileWrapper,
+    transaction_type: &'a api_enums::TransactionType,
+    key_store: hyperswitch_domain_models::merchant_key_store::MerchantKeyStore,
+}
+#[cfg(all(
+    feature = "v2",
+    feature = "routing_v2",
+    feature = "business_profile_v2"
+))]
+impl<'a> MerchantDefaultConfigUpdate<'a> {
+    async fn update_merchant_default_config(&self) -> RouterResult<()> {
```

Comment:
```suggestion
    async fn update_merchant_default_routing_config(&self) -> RouterResult<()> {
```

Reply:
And the function name should express what it does, like use-case-wise, eg: update_default_fallback_routing_if_routable_connector_exists.